docs/build/references/platform-transaction-serialization.md

```diff
@@ -891,6 +891,196 @@ Let’s make an unsigned add delegator tx that uses the inputs and outputs from
+  0x61, 0x76, 0x6d, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+
+  // fxids
+  // num fxids 
+  0x00, 0x00, 0x00, 0x01, 
+  // fxid 
+  0x73, 0x65, 0x63, 0x70, 0x32, 0x35, 0x36, 0x6b, 
+  0x31, 0x66, 0x78, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+
+  // genesis data len 
+  0x00, 0x00, 0x00, 0xb0, 
+  // genesis data 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x0e, 
+  0x41, 0x73, 0x73, 0x65, 0x74, 0x41, 0x6c, 0x69, 
+  0x61, 0x73, 0x54, 0x65, 0x73, 0x74, 0x00, 0x00, 
+  0x05, 0x39, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x1b, 0x66, 0x72, 
+  0x6f, 0x6d, 0x20, 0x73, 0x6e, 0x6f, 0x77, 0x66, 
+  0x6c, 0x61, 0x6b, 0x65, 0x20, 0x74, 0x6f, 0x20, 
+  0x61, 0x76, 0x61, 0x6c, 0x61, 0x6e, 0x63, 0x68, 
+  0x65, 0x00, 0x0a, 0x54, 0x65, 0x73, 0x74, 0x20, 
+  0x41, 0x73, 0x73, 0x65, 0x74, 0x00, 0x04, 0x54, 
+  0x45, 0x53, 0x54, 0x00, 0x00, 0x00, 0x00, 0x01, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 
+  0x00, 0x00, 0x00, 0x07, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x01, 0xfb, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 
+  0x00, 0x00, 0x00, 0x01, 0x3c, 0xb7, 0xd3, 0x84, 
+  0x2e, 0x8c, 0xee, 0x6a, 0x0e, 0xbd, 0x09, 0xf1, 
+  0xfe, 0x88, 0x4f, 0x68, 0x61, 0xe1, 0xb2, 0x9c, 
+
+  // type id (Subnet Auth) 
+  0x00, 0x00, 0x00, 0x0a, 
+  // num address indices 
+  0x00, 0x00, 0x00, 0x01, 
+  // address index 
+  0x00, 0x00, 0x00, 0x00, 
+
+  // num creds 
+  0x00, 0x00, 0x00, 0x02, 
+  // cred 1
+  // type id 
+  0x00, 0x00, 0x00, 0x09, 
+  // num sigs 
+  0x00, 0x00, 0x00, 0x01, 
+  // sig 1 
+  0xf4, 0xea, 0xb4, 0x8f, 0x0d, 0xa8, 0x24, 0x6b, 
+  0x62, 0xfb, 0x29, 0xa0, 0x14, 0x1b, 0x65, 0x41, 
+  0x99, 0x17, 0x64, 0x40, 0x83, 0x2e, 0x2b, 0xc8, 
+  0xed, 0x01, 0x10, 0x25, 0x47, 0x7d, 0x3d, 0xa7, 
+  0x17, 0xb5, 0xe6, 0xe1, 0xe7, 0xf2, 0x2f, 0x74, 
+  0xb4, 0x39, 0x02, 0x91, 0xe9, 0x3f, 0x93, 0x76, 
+  0xf8, 0x60, 0xa1, 0x2c, 0x15, 0xc9, 0xf1, 0x02, 
+  0x91, 0xc3, 0x4e, 0x27, 0x85, 0x7c, 0x6d, 0x20, 
+  0x01, 
+  // cred 2
+  // type id 
+  0x00, 0x00, 0x00, 0x09, 
+  // num sigs 
+  0x00, 0x00, 0x00, 0x01, 
+  // sig 2 
+  0xf4, 0xea, 0xb4, 0x8f, 0x0d, 0xa8, 0x24, 0x6b, 
+  0x62, 0xfb, 0x29, 0xa0, 0x14, 0x1b, 0x65, 0x41, 
+  0x99, 0x17, 0x64, 0x40, 0x83, 0x2e, 0x2b, 0xc8, 
+  0xed, 0x01, 0x10, 0x25, 0x47, 0x7d, 0x3d, 0xa7, 
+  0x17, 0xb5, 0xe6, 0xe1, 0xe7, 0xf2, 0x2f, 0x74, 
+  0xb4, 0x39, 0x02, 0x91, 0xe9, 0x3f, 0x93, 0x76, 
+  0xf8, 0x60, 0xa1, 0x2c, 0x15, 0xc9, 0xf1, 0x02, 
+  0x91, 0xc3, 0x4e, 0x27, 0x85, 0x7c, 0x6d, 0x20, 
+  0x01, 
+]
```

Comment:
Should we include the signatures here? This is supposed to be an unsigned tx example, no?

Reply:
Good catch. Removed.